pkgs/development/python-modules/fiat/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchurl
+, python3Packages
+, dolfin
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "fiat";
+  inherit (dolfin) version;
+
+  src = fetchurl {
+    url = "https://bitbucket.org/fenics-project/fiat/downloads/fiat-${version}.tar.gz";
+    sha256 = "1sbi0fbr7w9g9ajr565g3njxrc3qydqjy3334vmz5xg0rd3106il";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ numpy six sympy ];
+
+  checkInputs = with python3Packages; [ pytest ];
```

Comment:
python3Packages is not allowed in python modules. Please use the same solution as shown below.